integration-tests/src/test/kotlin/org/stellar/anchor/platform/PlatformTests.kt

```diff
@@ -92,3 +95,113 @@ fun testHealth() {
   assertEquals(streams[0]["thread_terminated"], false)
   assertEquals(streams[0]["stopped"], false)
 }
+
+fun testSep31UnhappyPath() {
```

Comment:
Awesome e2e test!

Reply:
This is what we had been wanting. Love to see these tests.